src/defaultsettings.cpp

```diff
@@ -565,7 +566,10 @@ void set_default_settings()
 	settings->setDefault("active_block_range", "2");
 	settings->setDefault("viewing_range", "50");
 	settings->setDefault("leaves_style", "simple");
+	// Note: OpenGL ES 2.0 is not guaranteed to provide depth textures,
```

Comment:
well, OpenGL ES 2.0 is even guaranteed not to provide depth textures, strictly speaking